docs/getting-started/prerequisites.md

```diff
@@ -0,0 +1,237 @@
+# Setting up the prerequisites
+
+This guideline includes the steps to install the prerequisites with specified versions for the Carbyne Stack. All installation procedures are done with the Terminal application of Ubuntu. **Please open the Terminal application before you proceed.**
+
+## Prerequisites
+
+This part of the tutorial is developed and tested with Ubuntu 20.04. Please refer to this [link](https://ubuntu.com/tutorials/install-ubuntu-desktop) for Ubuntu installation steps.
+
+## Platform Setup Prerequisites
+
+Software to be installed:
+
+- [go](https://go.dev/doc/install)
+- [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) v20.10.6
+- [Kind](https://kind.sigs.k8s.io/) v0.11.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) v1.21.1
+- [Helm](https://helm.sh/docs/intro/install/) v3.7.1
+
+### go
+
+The go language is a prerequisite for the Docker Engine. In this guideline go 1.18 will be installed.
```

Comment:
From my understanding this is not the case. You can install Docker Engine without golang being available. Could be that I am wrong, however. Can you explain where and why go is required?

Reply:
Sorry, this is my mistake. We need this for Kind, not Docker Engine. It is now corrected.